test/Atmos/EDMF/closures/surface_functions.jl

```diff
@@ -58,7 +57,7 @@ function subdomain_surface_values(
     oblength = turbconv.surface.obukhov_length
     ustar = turbconv.surface.ustar
 
-    unstable = oblength < 0
+    unstable = oblength < -eps(FT)
```

Comment:
do we consider 'oblength=0' a stable case or do we have a neutral case as well ?

Reply:
In reality, oblength-> 0 is not possible because the numerator is nonzero. We use oblength=0 for now because we only modify the EDMF scheme with oblength <0. We will have to pass this from the SurfaceFluxes.jl module in the end.